proto/grpc_route.proto

```diff
@@ -31,3 +31,8 @@ message GrpcFailureInjector {
   // ALL requests are failed.
   io.linkerd.proxy.http_route.Ratio ratio = 3;
 }
+
+// A list of gRPC response status codes
+message GrpcStatuses {
+  repeated uint32 codes = 1;
```

Comment:
I don't think there's much harm in having the API be overly permissive here.  Declaring a non-existent code as a failure shouldn't cause any problems and we can add more guardrails at the user interface (i.e. resource validation).

Reply:
I think we should add classifiers independently from circuit breakers. They are not coupled as far as the API change goes.

Reply:
That's a good point @olix0r, let's move the response classification to a separate branch? In particular, it occurred to me that it might be desirable for gRPC classifiers to _also_ include HTTP status ranges for cases where a gRPC service returns an HTTP error code...